homeassistant/components/anova/sensor.py

```diff
@@ -49,11 +50,15 @@ class AnovaSensorEntityDescription(
     AnovaSensorEntityDescription(
         key="state",
         translation_key="state",
+        device_class=SensorDeviceClass.ENUM,
+        options=[state.name for state in AnovaState],
```

Comment:
What were the possible states before? Is this a breaking change?
From the tests it looks like this is also a breaking change

Reply:
Added it above.